howto/application/update.md

```diff
@@ -84,7 +84,7 @@ In addition, each version has a status which indicates the status of the bootstr
 <a name="publish-application-versions"></a>
 ## Publish application versions
 
-The most important part of an application version is the `published` field. If a version is marked as published, it is accessible to users of Anbox Cloud. Generally when launching containers by using the AMS REST API, if no specific application version is given, the last published version of an application is used to create the container.
+The most important part of an application version is the `published` field. If a version is marked as published, it is available to launch and use. Generally when launching containers by using the AMS REST API, if no specific application version is given, by default, the last published version of an application is used to create the container.
```

Comment:
Shouldn't it be 'latest'?
```suggestion
The most important part of an application version is the `published` field. If a version is marked as published, it is available to launch and use. Generally when launching containers by using the AMS REST API, if no specific application version is given, by default, the latest published version of an application is used to create the container.
```

What do you think?

Reply:
Both are fine but in the context of versions, I would agree that 'latest' can be favoured.